### PR TITLE
added support for postgresql

### PR DESCRIPTION
**Pull request by barrachri**
Addedd support for postgresql

You need to use os env for sqlite/postgres, user and password

### CODE REVIEW

File: pycon/settings.py [hunk 1]
```diff
@@ -25,7 +25,21 @@
...
-DATABASES = {
+DATABASE_TYPE = os.environ.get('DATABASE')
+
+if DATABASE_TYPE == "postgres":
+    user = os.environ.get('DATABASE_USER')
+    password = os.environ.get('DATABASE_PASSWORD')
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'ep2015_testing',
```
Comment (matrixise) [inline pycon/settings.py]: in this case, the database is hardcoded.

via the CLI of manage.py , you can specify the settings for the production and the development.

File: p3/views/profile.py [hunk 2]
```diff
@@ -161,3 +162,31 @@ def p3_account_spam_control(request):
         if form.is_valid():
             form.save()
     return render(request, "assopy/profile_spam_control.html", ctx)
```
Comment (matrixise) [inline p3/views/profile.py:164]: what's assopy?

File: pycon/settings.py (hunk 1 above)
Comment (barrachri) [inline pycon/settings.py]: We have only one setting right now.
We can add another env var (DATABASE_NAME) to set db name.

File: p3/views/profile.py (hunk 2 above)
Comment (barrachri) [inline p3/views/profile.py:164]: A library I think.

Comment (fpliger) [inline p3/views/profile.py:164]: it's the django app that takes care of user accounts, invoices, payments, orders, etc...

Comment (matrixise) [inline p3/views/profile.py:164]: https://github.com/pythonitalia/assopy

File: pycon/settings.py (hunk 1 above)
Comment (matrixise) [inline pycon/settings.py]: I agree with you about one setting, but I have the experience where we can forget to change the settings during the deployment in production. but ok.

File: p3/views/profile.py [hunk 3]
```diff
@@ -161,3 +162,31 @@ def p3_account_spam_control(request):
...
+        # check if assopy user have already been created for this user
+        asso_user = user.assopy_user
+    except amodels.User.DoesNotExist:
+        # create it if not...s
+        log.debug('the current user "%s" will become an assopy user', user)
+        asso_user = amodels.User(user=user)
+        asso_user.save()
+
+    # same for conference profile...
+    profile = cmodels.AttendeeProfile.objects.getOrCreateForUser(user)
```
Comment (fpliger) [inline p3/views/profile.py:192]: @barrachri just out of curiosity.. is this from a branch of mine or something? I remember writing the same thing.. :-)

File: pycon/settings.py (hunk 1 above)
Comment (fpliger) [inline pycon/settings.py]: +1 on adding an env var in this PR

that said I find having all those mandatory env vars quite annoying... I'd prefer have a "default" set to a sqlite db type (somethign like `DATABASE_TYPE = os.environ.get('DATABASE', 'sqlite')) 
Maybe something for a future PR/discussion

File: p3/views/profile.py (hunk 3 above)
Comment (barrachri) [inline p3/views/profile.py:192]: You remember right....one of my commits is yours. 
I think something went wrong with my merge/push.

File: pycon/settings.py (hunk 1 above)
Comment (barrachri) [inline pycon/settings.py]: There is a "default" set, from line 41.

**Comment by barrachri** [comment]
everything's fine now !

**Pull request merged**